### PR TITLE
Pre publication

### PR DESCRIPTION
**Pull request by iSkoog**
Fixes # (issue)

## What Changed

1. Please include a summary list changes and which issue is fixed. Examples:
2. Fixed bug #1 by correcting a typo
3. Refactored the flux capacitor to be more energy efficient

## Limitations

1. Please include any major limitations. Examples
2. None
3. Did not have the proper test harness to test the flux capacitor outside of normal driving conditions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,53 @@
...
-
-At the end of the project, the visualization tools will demonstrate how attack flows will help security leaders explain defensive posture to executives, help defenders understand lessons-learned in an incident, and help red-teamers easily compose realistic adversary emulation scenarios.
-
-Resources:
-* [Attack Flow Specification](/docs/attack-flow-schema.md)
-* [Attack Flow JSON Schema](/schema/attack-flow-2022-01-05-draft.json)
-* [Attack Flow Example](/schema/attack-flow-example.json)
+The Attack Flow project helps defenders move from tracking adversary behaviors individually to the sequence of techniques adversaries use to achieve their goals. Understanding the context within those sequences, as well as the relationships among them, enables additional defensive capabilities that make defenders much more effective. The project seeks to demonstrate how attack flows can explain defensive posture to executives, aid defenders understand lessons-learned from an incident, and support red-teamers easily compose realistic adversary emulation scenarios.
```
Comment (mehaase) [inline README.md:4]: > aid defenders understand lessons-learned

Missing a comma or a word here?

> support red-teamers easily compose

Also here.

**Review by MarkDavidson — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,22 +1,53 @@
...
+    1. File → Save Attack Flow: saves the file with `*.afd` extension. This file is suitable for opening up
+       for editing in the Attack Flow Designer.
+    2. File → Publish Attack Flow: saves a `*.json` file that conforms to the Attack Flow specification; this
+       file is interoperable with other Attack Flow tools.
+
+## Getting Involved
+There are several ways that you can get involved with this project and help advance threat-informed defense: 
+- **Review the schema, use the builder, and tell us what you think.**  We welcome your review and feedback on the data model and our methodology.
+- **Help us prioritize additional example flows to create.** Let us know what examples you would like to turned into an Attack Flow. Your input will help us prioritize how we expand our corpus.
+- **Share your use cases.** We are interested in developing additional tools and resources to help the community understand and make threat-informed decisions in their risk management programs. If you have ideas or suggestions, we consider them as explore additional research projects.
```
Comment (mehaase) [inline README.md:42]: > consider them as explore

Missing a "we".

**Review by mehaase — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (iSkoog) [reply README.md]: Should be changed to make it more readable - added a couple 'to's

File: README.md (hunk 2 above)
Comment (iSkoog) [reply README.md]: added

**Review by mehaase — APPROVED** [review]

**Pull request merged**